ui.apps/src/main/content/jcr_root/apps/etoolbox-authoring-kit/allowed-children/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+          jcr:primaryType="cq:ClientLibraryFolder"
+          categories="[cq.authoring.allowed-children]"
```

Comment:
Name the category `cq.authoring.eak.policies` There can be more than just allowed children.